<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>708</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1366</width>
    <height>708</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1366</width>
    <height>708</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>311</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="lab0">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lab1">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>111</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QComboBox" name="cb0"/>
      </item>
      <item>
       <widget class="QComboBox" name="cb1"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>91</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QPushButton" name="pb0">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb1">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>1321</width>
      <height>521</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Robot A</string>
     </attribute>
     <widget class="QCustomPlot" name="plot0" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>761</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="plot0L" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>491</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="plot0R" native="true">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>261</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="plot0V" native="true">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>10</y>
        <width>531</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Robot B</string>
     </attribute>
     <widget class="QCustomPlot" name="plot1" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>761</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="plot1L" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>491</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="plot1R" native="true">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>261</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="plot1V" native="true">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>10</y>
        <width>531</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Robot C</string>
     </attribute>
     <widget class="QCustomPlot" name="plot2" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>761</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="plot2R" native="true">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>261</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="plot2L" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>491</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="plot2V" native="true">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>10</y>
        <width>531</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>124</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>RadioButton</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>RadioButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_3">
        <property name="text">
         <string>RadioButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_4">
        <property name="text">
         <string>RadioButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>111</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pb4">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>741</width>
      <height>141</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Robot A Data Table Format</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>731</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Robot B Data Table Format</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>731</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Robot C Data Table Format</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>731</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSwarm_GUI">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionsave_to_csv_file"/>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_us"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionShow_graph_2"/>
    <addaction name="separator"/>
    <addaction name="actionInitial_position"/>
   </widget>
   <addaction name="menuSwarm_GUI"/>
   <addaction name="menuOptions"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionsave_to_csv_file">
   <property name="text">
    <string>Save to csv file</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout_us">
   <property name="text">
    <string>About us</string>
   </property>
  </action>
  <action name="actionInitial_position">
   <property name="text">
    <string>Initial position setting</string>
   </property>
  </action>
  <action name="actionShow_graph_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show all graph</string>
   </property>
  </action>
  <action name="actionShow_gas_concentration_graph">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show gas concentration graph</string>
   </property>
  </action>
  <action name="actionShow_PWM_graph">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show PWM graph</string>
   </property>
  </action>
  <action name="actionShow_LiDAR_view_graph">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show LiDAR view graph</string>
   </property>
  </action>
  <action name="actionShow_robot_trajectory_graph">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show robot trajectory graph</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
